src/Database/Driver/Sqlserver.php

```diff
@@ -64,8 +64,14 @@ public function connect()
             return true;
         }
         $config = $this->_config;
+
+        // PDO::ATTR_PERSISTENT is not allowed by SQL Server PHP drivers.
+        // See: https://github.com/Microsoft/msphpsql/issues/65
```

Comment:
You could move these two lines above and add an ``@link`` to the doc block to increase visibilty in the API docs for this issue.